app/services/instagram/send_on_instagram_service.rb

```diff
@@ -96,4 +100,12 @@ def last_incoming_message
   def config
     Facebook::Messenger.config
   end
+
+  def merge_human_agent_tag(params)
+    return unless ENV.fetch('ENABLE_MESSENGER_CHANNEL_HUMAN_AGENT', false)
```

Comment:
```suggestion
    return params unless ENV.fetch('ENABLE_MESSENGER_CHANNEL_HUMAN_AGENT', false)
```

let's have this as an installation config rather than an environment variable. since we can update it without restarting the servers.